frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/row_detail.dart

```diff
@@ -43,147 +45,100 @@ class RowDetailPage extends StatefulWidget with FlowyOverlayDelegate {
 }
 
 class _RowDetailPageState extends State<RowDetailPage> {
-  final padding = const EdgeInsets.symmetric(
-    horizontal: 40,
-    vertical: 20,
-  );
-
   @override
   Widget build(BuildContext context) {
     return FlowyDialog(
       child: BlocProvider(
         create: (context) {
-          final bloc = RowDetailBloc(
-            dataController: widget.dataController,
-          );
-          bloc.add(const RowDetailEvent.initial());
-          return bloc;
+          return RowDetailBloc(dataController: widget.dataController)
+            ..add(const RowDetailEvent.initial());
         },
-        child: Padding(
-          padding: padding,
-          child: Column(
-            children: [
-              const _Header(),
-              Expanded(
-                child: _PropertyColumn(
-                  cellBuilder: widget.cellBuilder,
-                  viewId: widget.dataController.viewId,
-                ),
+        child: ListView(
+          children: [
+            // const SizedBox(height: 100),
+            // const Divider(height: 1.0),
+            IntrinsicHeight(
```

Comment:
Is it possible to avoid using IntrinsicHeight here?

From [docs](https://api.flutter.dev/flutter/widgets/IntrinsicHeight-class.html)
> This class is relatively expensive, because it adds a speculative layout pass before the final layout phase. Avoid using it where possible. In the worst case, this widget can result in a layout that is O(N²) in the depth of the tree.

Reply:
Hello @a-wallen! I tried as much as possible to avoid `IntrinsicHeight` but I really couldn't figure out a more eloquent solution. If I simply remove it, the `VerticalDivider` inside the `Row` will get a height of zero making it disappear. If I do `CrossAxxiAlignment.stretch`, the `Column`s get unbounded height and throw exceptions. I've also tried experimenting with drawing left and right borders on the columns, but the heights of the columns aren't equal either. Please let me know if you have any suggestions to do this properly 🙏

Reply:
No worries. We can always fix it if it becomes an issue in the future.